src/backend/electron/main.ts

```diff
@@ -163,6 +163,27 @@ void app.whenReady().then(() => {
   });
 });
 
+// リモートコンテンツからのセッション権限リクエストを全て拒否
+void app.whenReady().then(() => {
+  session.defaultSession.setPermissionRequestHandler(
+    (webContents, permission, callback, { requestingUrl }) => {
+      const parsedUrl = new URL(webContents.getURL());
+      const parsedRequestingUrl = new URL(requestingUrl);
+      let isAppUrl: boolean;
+      if (import.meta.env.VITE_DEV_SERVER_URL != undefined) {
```

Comment:
import.meta.env.DEVのほうが適していると思います。
一応これはDead Code Eliminationで消えると思いますが、まぁ一応...

```suggestion
      if (import.meta.env.DEV) {
```

Reply:
型定義上`import.meta.env.VITE_DEV_SERVER_URL != undefined`は消せないのですよね。
そうなると冗長な気がします。

Reply:
ん～、となるとassertNotNull(import.meta.env.VITE_DEV_SERVER_URL)を使ってもいいかも？DEVかつVITE_DEV_SERVER_URL == undefinedはunreachableなのでそれを防げますし。

Reply:
たぶん要するにこういうことですよね：
```suggestion
      if (import.meta.env.DEV != undefined) {
        assertNonNullable(import.meta.env.VITE_DEV_SERVER_URL)
```

assertNonNullableはこれ↓
https://github.com/VOICEVOX/voicevox/blob/d2a9ebf67267b1ce79a6840fe0519193a16dcc04/src/type/utility.ts#L14C17-L14C34